packages/manager/modules/account/src/user/dashboard/user-dashboard.constants.js

```diff
@@ -34,12 +38,23 @@ export const USER_DASHBOARD_SHORTCUTS = [
       path: '#/billing/payment/method/add',
     },
     isAvailable: (user) => !user.enterprise,
+    tracking_name_suffix: 'add-payment-method',
   },
   {
     key: 'ADD_PAYMENT_METHOD',
     href: 'https://billing.us.ovhcloud.com/login',
     isAvailable: (user) => user.enterprise,
     isExternal: true,
+    tracking_name_suffix: 'add-payment-method',
+  },
+  {
+    key: 'ALL_AGREEMENTS',
+    url: {
+      baseURL: 'dedicated',
+      path: '#/billing/autorenew/agreements',
+    },
```

Comment:
You should use the new account app
```suggestion
    url: {
      baseURL: 'billing',
      path: '#/autorenew/agreements',
    },
```